challenge_3/cpp/blakebarnes00/src/main.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+void print_array(std::vector<int> array)
```

Comment:
No worries! A reference is a... nicer pointer? The variable access would work exactly the same way in your function so no other code would need to be changed. The compiler would just be utilizing a pointer to the vector under the covers instead of making a local copy every time this gets called.